Dockerfile

```diff
@@ -150,6 +150,7 @@ ENV PEGA_SEARCH_URL=
 
 # Configure hazelcast. By default, hazelcast runs in embedded mode.
 ENV HZ_CLIENT_MODE=false \
+    HZ_VERSION=v4 \
```

Comment:
Why the default value set to `v4`?

Reply:
As discussed over the call , v4 is default as we had in pega helm chart before.
This value will be overridden by Pega Helm chart accordingly based on Pega version.

Reply:
After discussing with @vnihal72 , I have learned that earlier this default value is set in helm charts, same is moved to docker,